config/locales/sk.yml

```diff
@@ -40,18 +40,18 @@ sk:
     sidebars:
       filters: "Filtre"
     pagination:
-      empty: "Nenájdený."
-      one: "Zobrazená  <b>1</b> položka"
-      one_page: "Počet zobrazených položiek %{n}"
-      multiple: "<b>%{from}&nbsp;-&nbsp;%{to}</b> z <b>%{total}</b>"
-      multiple_without_total: "<b>%{from}&nbsp;-&nbsp;%{to}</b>"
+      empty: "Nenájdený %{model}."
+      one: "Zobrazená  <b>1</b> %{model}"
```

Comment:
If the model is of female gender (položka = item), it is correct.
But, if e.g. the model is "auto", then pagination.one should be "Zobrazené 1 auto"